federation-2/router-bridge/js-src/plan_worker.ts

```diff
@@ -191,10 +191,21 @@ async function run() {
         }
       } catch (e) {
         logger.warn(`an error happened in the worker runtime ${e}\n`);
+
+        const unexpectedError = {
+          name: e.name || "unknown",
+          message: e.message || "",
+          extensions: {
+            code: "QUERY_PLANNING_FAILED",
+            exception: {
+              stacktrace: [e.toString().split(/\n/)],
```

Comment:
Thoughts on just creating an actual `GraphQLError` here (which accepts I think everything besides `name` already) rather than creating a object that is roughly shaped like the `GraphQLError`?

I'm curious what wouldn't work about that?